morpheus/_lib/src/stages/triton_inference.cpp

```diff
@@ -397,8 +393,8 @@ bool InferenceClientStage::is_default_grpc_port(std::string &server_url)
 }
 
 // ************ InferenceClientStageInterfaceProxy********* //
-std::shared_ptr<InferenceClientStage> InferenceClientStageInterfaceProxy::init(
-    neo::Segment &parent,
+std::shared_ptr<srf::segment::Object<InferenceClientStage>> InferenceClientStageInterfaceProxy::init(
+    srf::segment::Builder &parent,
```

Comment:
Bit of a nit pick, but could we call this `builder` or something, `parent` doesn't really describe what the variable is.

Reply:
I've renamed all parents to builder. Resolving all similar comments